mlflow/openai/__init__.py

```diff
@@ -693,8 +736,11 @@ def _predict_chat(self, data, params):
         from mlflow.openai.api_request_parallel_processor import process_api_requests
 
         _validate_model_params(self.task, self.model, params)
+        envs = _validate_params_and_envs(params, self.envs)
         messages_list = self.format_completions(self.get_params_list(data))
-        requests = [{**self.model, **params, "messages": messages} for messages in messages_list]
+        requests = [
+            {**self.model, **params, "messages": messages, **envs} for messages in messages_list
```

Comment:
```suggestion
            {**self.model, "messages": messages, **envs, **params} for messages in messages_list
```

Can we move `params` to the last to override envs?